test/functional/e2e/registration.e2e-spec.ts

```diff
@@ -0,0 +1,73 @@
+import {
+  getUser,
+  removeUserMutation,
+} from '../user-management/user.request.params';
+
+import puppeteer from 'puppeteer';
+
+import RegistrationPage from './registration-page-object';
+import { uniqueId } from '@test/utils/mutations/create-mutation';
+
+let userId;
+const email = `mail-${uniqueId}@alkem.io`;
+const password = 'test45612%%$';
+const firstName = 'testFN';
+const lastName = 'testLN';
+const userFullName = firstName + ' ' + lastName;
+const registrationPage = new RegistrationPage();
+
+describe('Registration smoke tests', () => {
+  let browser: puppeteer.Browser;
+  let page: puppeteer.Page;
+  beforeAll(async () => {
+    browser = await puppeteer.launch({
+      headless: true,
+      // slowMo: 10,
+    });
+  });
+
+  afterEach(async () => {
+    const client = await page.target().createCDPSession();
+    await client.send('Network.clearBrowserCookies');
+  });
+
+  afterAll(async () => {
+    await browser.close();
+  });
+
+  describe('add task to the list', () => {
+    beforeEach(async () => {
+      page = await browser.newPage();
+      await page.goto(process.env.ALKEMIO_BASE_URL + '/auth/registration');
+    });
+
+    test('User registers successfully', async () => {
+      await registrationPage.register(
+        page,
+        email,
+        password,
+        firstName,
+        lastName,
+        true
+      );
+      expect(
+        await registrationPage.verifyAuthenticatedUserAvatar(page)
+      ).toContain(userFullName);
+
+      const requestUserData = await getUser(email);
+      userId = requestUserData.body.data.user.id;
+      await removeUserMutation(userId);
```

Comment:
do we need a check if it's deleted sucessfully? what do you think?

Reply:
No. This call is tested in isolation in a separate test suite, which is part of integration tests.